dashboard/test/ui/features/applab/sharingFromScriptLevel.feature

```diff
@@ -1,6 +1,6 @@
 @dashboard_db_access
 @as_student
-Feature: App Lab Scenarios
+Feature: sharingFromScriptLevel
```

Comment:
We seem to usually use friendly names, but from discussion this seems like an interesting trend to try starting.